Decred Wallet/Features/Wallets/Wallets.storyboard

```diff
@@ -762,7 +762,7 @@
                         <constraints>
                             <constraint firstItem="ggu-IF-Ehw" firstAttribute="leading" secondItem="ArT-Pb-cdS" secondAttribute="leading" id="1mp-JX-0HT"/>
                             <constraint firstItem="ggu-IF-Ehw" firstAttribute="trailing" secondItem="ArT-Pb-cdS" secondAttribute="trailing" id="VKB-o2-RFR"/>
-                            <constraint firstItem="ArT-Pb-cdS" firstAttribute="bottom" secondItem="ggu-IF-Ehw" secondAttribute="bottom" id="mHX-Tv-9CH"/>
+                            <constraint firstAttribute="bottom" secondItem="ggu-IF-Ehw" secondAttribute="bottom" id="mHX-Tv-9CH"/>
```

Comment:
I rather recommend setting the bottom constraint of `stackView` to `safeArea`. 
Delete line 748 and put this line after line 765:

```<constraint firstItem="ViB-dv-2vV" firstAttribute="bottom" secondItem="ArT-Pb-cdS" secondAttribute="bottom" id="nwZ-Jc-YUw"/>```

Reply:
Perfect! I was wondering how to achieve that perfect bottom constraint for different device configs. This is it.